flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -168,8 +167,9 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         val name = sdk["name"] as? String
         val version = sdk["version"] as? String
         if (name != null && version != null) {
+          val sdkVersion = SdkVersion(name, version)
           options.setSentryClientName(name)
-          options.setSdkVersion(version)
+          options.setSdkVersion(sdkVersion)
```

Comment:
@denrase check this.

Reply:
I'm getting an issue when trying to import this

<img width="1350" alt="Bildschirm­foto 2023-01-16 um 10 45 49" src="https://user-images.githubusercontent.com/3984453/212647721-11551fac-0e1c-4077-b86d-1e2c1541c32f.png">

Using `PrivateSentrySDKOnly.setSdkName(name, andVersionString: version)` instead.

Reply:
Nevermind, this is kotlin.